lib/AST/TypeCheckRequests.cpp

```diff
@@ -1368,6 +1368,38 @@ void AssociatedConformanceRequest::cacheResult(
   conformance->setAssociatedConformance(index, assocConf);
 }
 
+//----------------------------------------------------------------------------//
+// ConformanceIsolationRequest computation.
+//----------------------------------------------------------------------------//
+bool ConformanceIsolationRequest::isCached() const {
+  // We only want to cache for global-actor-isolated conformances. For
+  // everything else, which is nearly every conformance, this request quickly
+  // returns "nonisolated" so there is no point in caching it.
```

Comment:
This is what split caching is for

Reply:
Ahh yes, so it is! Thanks for the reminder---I don't think I've used this feature before.